### PR TITLE
Fixes #5 - Add support for non-global pageId - allow specifying it in ancestor's attribute/property

### LINKED ISSUE #5: Add support for content from multipe CMS pages in single document
Using the global variable `CMSCurrentPageId` limits us to properties from just a single CMS page.

Add support for it, e.g. by using a property/attribute on the closest ancestor.

### PR DESCRIPTION
**Pull request by slawomir-brzezinski-at-interxion**
Implemented using the generic [AqoviaElements/properties-from-ancestor-behavior](https://github.com/AqoviaElements/properties-from-ancestor-behavior)

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
   "homepage": "https://github.com/AqoviaElements/epi-cms-property",
   "ignore": [],  
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.4.0"
+    "polymer": "Polymer/polymer#^1.4.0",
+    "properties-from-ancestor-behavior": "~1.0"
```
Comment (ma499) [inline bower.json]: Intentional to use ~ instead of ^ as version specifier?

Comment (slawomir-brzezinski-at-interxion) [reply bower.json]: Yes. I'm treating the decision that you and I recently made for the internal `inventory-overview` package (to start using SemVer with `~`), as applying to the team that authors the package, which is the same here.  

My idea is to change it to `^` once we have evidence that most of team has been marking breaking changes correctly.

Don't mind if you'd rather start trustful. I can change it.

Comment (ma499) [reply bower.json]: Well internally the major version number is not controlled in source. Here it's in source, so I think we should take it upon ourselves to always consider breaking changes - and have adequate tests to identify them. But I see that it merits discussion :-)

Comment (slawomir-brzezinski-at-interxion) [reply bower.json]: The change I recently made to `inventory-overview` actually moves the control over major.minor to source.

But nevertheless, after more discussing, I agree that for public components there will probably be more caution in the team about these numbers, so I'm changing it to a carrot ;)

**Review by timrog1 — APPROVED** [review]

**Review by ma499 — APPROVED** [review]

**Pull request merged**